src/test/java/io/github/bonigarcia/wdm/test/OperaTest.java

```diff
@@ -35,8 +36,14 @@
  */
 public class OperaTest extends BaseBrowserTst {
 
+	static boolean headlessMode = "true".equals(System.getProperty("headless"));
```

Comment:
WebDriverManager uses the library `config` by TypeSafe to handle properties. Therefore, I think it's better to continue using this approach to this property (BTW I don't like the name `headless`, since it is only used for Opera). I propose to use the following:

`boolean operaTests = WdmConfig.getBoolean("wdm.operaTests");`

This new key `wdm.operaTests` should appear in the configuration file ([application.properties](https://github.com/bonigarcia/webdrivermanager/blob/master/src/main/resources/application.properties)), and also to the [README](https://github.com/bonigarcia/webdrivermanager/blob/master/README.md) (to explain the new key in the _advanced_ section).